plugins/out_prometheus_remote_write/remote_write.h

```diff
@@ -33,6 +33,20 @@ struct prometheus_remote_write_context {
     char *http_user;
     char *http_passwd;
 
+    /* AWS Auth */
+#ifdef FLB_HAVE_AWS
+    int has_aws_auth;
+    char *aws_region;
+    char *aws_sts_endpoint;
+    struct flb_aws_provider *aws_provider;
+    struct flb_aws_provider *base_aws_provider;
+    /* tls instances can't be re-used; aws provider requires a separate one */
+    struct flb_tls *aws_tls;
+    /* one for the standard chain provider, one for sts assume role */
+    struct flb_tls *aws_sts_tls;
+    char *aws_session_name;
```

Comment:
These variables can be removed if flb_managed_chain_provider_create() is used. Please see comment below.